scene/animation/tween.h

```diff
@@ -60,11 +61,13 @@ class PropertyTweener;
 class IntervalTweener;
 class CallbackTweener;
 class MethodTweener;
+class SubtweenTweener;
 
 class Tween : public RefCounted {
 	GDCLASS(Tween, RefCounted);
 
 	friend class PropertyTweener;
+	friend class SceneTree;
```

Comment:
Not needed anymore.

Reply:
Has been fixed and will be pushed shortly!